vuepress/zh/guide/pluralization.md

```diff
@@ -126,8 +128,8 @@ const messages = {
   }
 }
 ```
-Where the format is `0 things | things count ends with 1 | things count ends with 2-4 | things count ends with 5-9, 0 and teens (10-19)`.
-P.S. Slavic pluralization is a pain, you can read more about it [here](http://www.russianlessons.net/lessons/lesson11_main.php).
+格式在哪里 `0 东西 | 事情以结尾结束 1 | 事情以结尾结束 2-4 | 事情以结尾结束 5-9, 0 和青少年 (10-19)`.
+附言 斯拉夫多元化是困难的，您可以阅读有关它的更多信息 [这里](http://www.russianlessons.net/lessons/lesson11_main.php).
```

Comment:
@kazupon review this translate to chinese, and i rewrite word "pain" to "difficult" ok?